upbound_uxp/manifest.json

```diff
@@ -0,0 +1,71 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "f1427ca4-4b88-484a-a7c5-9cc66542b0d9",
+  "app_id": "upbound-uxp",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Collect and graph Upbound UXP metrics",
+    "title": "Upbound UXP",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Upbound",
+        "image_url": "images/upbound_uxp_logo.svg"
+      }
+    ],
+    "classifier_tags": [
+      "Category::AWS",
+      "Category::Azure",
+      "Category::Cloud",
+      "Category::Configuration & Deployment",
+      "Category::Containers",
+      "Category::Developer Tools",
+      "Category::Google Cloud",
+      "Category::Kubernetes",
+      "Category::Languages",
```

Comment:
```suggestion
```
This category refers to integrations for specific languages

Reply:
removed.